utils/static.js

```diff
@@ -748,7 +748,19 @@ var SLStatic = {
       dt.setDate(dt.getDate()+1);
     }
     return dt;
-  }
+  },
+
+  // HTML element IDs correspond to preference keys, localization strings, and
```

Comment:
It might make sense to put these at the top of this object, maybe right above the `logger` function just to group it with other statically defined constants.

Reply:
Will fix